### PR TITLE
IPU:for rhel7 to rhel8 upgrade path, add the installation of leapp from compose

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:2161657
The script supports installing leapp tool from compose.
Signed-off-by: MiriamDeng <mdeng@redhat.com>

### CODE REVIEW

**Comment by MiriamDeng** [comment]
(2/2) Host_RHEL.m9.u2.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.in_place_upgrade_legacy.without_rhsm_rhel7_to_rhel8.compose_install.q35: STARTED
 (2/2) Host_RHEL.m9.u2.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.in_place_upgrade_legacy.without_rhsm_rhel7_to_rhel8.compose_install.q35: PASS (5509.34 s)

**Comment by MiriamDeng** [comment]
@fbq815 @PaulYuuu could you help to review the patch ? Thanks a lot

File: qemu/tests/in_place_upgrade_legacy.py [hunk 1]
```diff
@@ -98,8 +98,9 @@ def run(test, params, env):
         upgrade_test.session = vm.reboot(upgrade_test.session)
         # repo_leapp_7 it's leapp tool's repo
         # repo_leppp_7_seed and ins_leapp_cmd, install leapp tool command
-        upgrade_test.run_guest_cmd(params.get("repo_leapp_7"))
-        upgrade_test.run_guest_cmd(params.get("repo_leppp_7_seed"))
+        if params.get("com_install") != "yes":
```
Comment (PaulYuuu) [inline qemu/tests/in_place_upgrade_legacy.py:101]:
```suggestion
        if params.get_boolean("com_install"):
```

Comment (MiriamDeng) [reply qemu/tests/in_place_upgrade_legacy.py]: Updated, thanks

**Comment by MiriamDeng** [comment]
(1/1) Host_RHEL.m8.u8.product_rhel.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.in_place_upgrade_legacy.without_rhsm_rhel7_to_rhel8.compose_install.q35: STARTED
 (1/1) Host_RHEL.m8.u8.product_rhel.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.in_place_upgrade_legacy.without_rhsm_rhel7_to_rhel8.compose_install.q35: PASS (7869.07 s)

**Review by PaulYuuu — APPROVED** [review]

**Review by fbq815 — APPROVED** [review]

**Pull request merged**